hopsworks-api/src/main/java/io/hops/hopsworks/api/project/RequestService.java

```diff
@@ -239,18 +243,26 @@ public Response requestJoin(RequestDTO requestDTO, @Context HttpServletRequest r
             getOwner().getLname() + ", \n\n"
             + user.getFname() + " " + user.getLname()
             + " wants to join a project you own. \n\n"
-            + "Project name: " + project.getName() + "\n"
-            + "Attached message: " + requestDTO.getMessageContent() + "\n"
-            + "After loging in to hopsworks go to : /project" + project.getId()
-            + " and go to members tab "
-            + "if you want to add this person as a member in your project. \n";
+            + "Project name: " + project.getName() + "\n";
+
+    if(requestDTO.getMessageContent() != null) {
```

Comment:
@robzor92 should we check for empty message as well here? as in `Strings.isNullorEmpty()`

Reply:
done